Makefile

```diff
@@ -39,16 +39,3 @@ configure:
 .PHONY: build doc test all install uninstall reinstall clean distclean configure
 
 # OASIS_STOP
```

Comment:
Where did the release target go? Is it no longer used? I didn't see it mentioned in the changes.